corehq/apps/domain/forms.py

```diff
@@ -1370,6 +1371,13 @@ def save(self, commit=True):
                 if not account_save_success:
                     return False
 
+                # changing a plan overrides future subscriptions
+                future_subscriptions = Subscription.objects.filter(
+                    subscriber=self.current_subscription.subscriber,
```

Comment:
this could error in the unlikely situation that ```current_subscription``` is None - might be safer / more future-proof to reference the domain's name directly